setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup, find_packages
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+requirements = [
+    'girder>=3.0.0a1'
```

Comment:
This can just be changed to `histomicsui`.  It is my preference to have these requirements in-line in the setup call rather than as a separate section here, mainly because most other python packages seem to do that.

Reply:
Is that because `histomicsui` will bring in the proper version of Girder itself?

Reply:
See cea46cd2c2d0a95d20b9c0acffd1cae51b6b056a

Reply:
> Is that because `histomicsui` will bring in the proper version of Girder itself?

Yes.